### ISSUE TITLE
should loss_scale be multiplied to the loss explicitly?

### ISSUE THREAD

**Issue by Jim-Song** [huggingface/pytorch-pretrained-BERT#552]
I noticed that in the run_swag.py, the following code is included

    if args.fp16 and args.loss_scale != 1.0:
        # rescale loss for fp16 training
        # see https://docs.nvidia.com/deeplearning/sdk/mixed-precision-training/index.html
        loss = loss * args.loss_scale

and in run_squad.py,  this is not included.
the optimizer in them are identity, so which is right?
